src/javaRosa.js

```diff
@@ -1393,6 +1393,9 @@ define([
             } else {
                 _this.__callOld();
             }
+            if (path.contains('casedb')) {
+                window.analytics.usage("Case Management", "Form Builder", "Click and Drag");
```

Comment:
I think both. @krsdimagi or @amsagoff can correct if that is wrong

On Wed, Mar 16, 2016 at 3:43 PM, Jonathan Emord <notifications@github.com>
wrote:

> In src/javaRosa.js
> <https://github.com/dimagi/Vellum/pull/625#discussion_r56401562>:
>
> > @@ -1393,6 +1393,9 @@ define([
> >              } else {
> >                  _this.__callOld();
> >              }
> > +            if (path.contains('casedb')) {
> > +                window.analytics.usage("Case Management", "Form Builder", "Click and Drag");
>
> Is this supposed to be only for labels or for labels and xpaths?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/Vellum/pull/625/files/cf5fbdbf288457b26a3cda2c3b33cb798eb90922#r56401562>
>